### ISSUE TITLE
Does Docker(Moby) still utilize libeseccomp-golang?

### ISSUE THREAD

**Issue by hazelnutsgz** [seccomp/libseccomp-golang#33]
In the latest version of [Docker](https://github.com/moby/moby), I found that the only occurrence of `libseccomp-golang` is `GetNativeArch()` https://github.com/moby/moby/blob/cfeba5af9ec522565fb527a78d97081cbb75ce8f/profiles/seccomp/seccomp.go#L66, 

I was wondering whether the other APIs like 
`func (f *ScmpFilter) Load() error`
`func (f *ScmpFilter) AddRule(call ScmpSyscall, action ScmpAction) error` 
**are still be utilized by Docker**?

Basically, I don't find the location in Docker's source code to load the seccomp filter to linux kernel. 

Thanks for your help~

 

Basicsally

**Comment by mheon** [comment]
Most Seccomp code lives in the OCI runtime - by default, `runc` (github.com/opencontainers/runc). You'll find a lot more uses of `libseccomp` there.

**Comment by hazelnutsgz** [comment]
> Most Seccomp code lives in the OCI runtime - by default, `runc` (github.com/opencontainers/runc). You'll find a lot more uses of `libseccomp` there.

In fact, the tricky thing here is that the code in official `runc` (https://github.com/opencontainers/runc/tree/master/libcontainer/seccomp) is different from the corresponding `libcontainer` in moby(https://github.com/moby/moby/tree/master/vendor/github.com/opencontainers/runc/libcontainer)

Technically, the `seccomp` folder was purged in moby project, that is what puzzled me.

**Comment by mheon** [comment]
That's just a vendor folder - `github.com/moby/moby` is using `github.com/opencontainers/runc` as a library, but is not using most of the Seccomp portions.

The vendored `runc` code in `github.com/moby/moby` is not used to launch containers. A `runc` binary, built directly from `github.com/opencontainers/runc`, will be invoked directly to do that.

**Comment by hazelnutsgz** [comment]
> That's just a vendor folder - `github.com/moby/moby` is using `github.com/opencontainers/runc` as a library, but is not using most of the Seccomp portions.
> 
> The vendored `runc` code in `github.com/moby/moby` is not used to launch containers. A `runc` binary, built directly from `github.com/opencontainers/runc`, will be invoked directly to do that.

Thanks, forgive my naive understanding of Golang, really appreciated your help. You can close the issue.
